File: level1/6 - Structures avancées/6 - Tarifs de l'auberge.c
//coded by cybereagle2001
#include <stdio.h>

int main()
{
   int age;
   int poids;
   int price;
   scanf("%d%d",&age,&poids);
   if (age == 60)
   {
      price=0;
      printf("%d",price);
   }
   else
   {
      if ( age < 10)
      {
         price = 5;
         printf("%d",price);
      }
      else 
      {
         if (poids >=20)
         {
            price = 30+10;
            printf("%d",price);
         }
         else
         {
            price= 30;
            printf("%d",price);
         }
      }
   }
}

File: java/level 1/6- Le grand voyage/6- Tarifs de l'auberge.java
/** coded by cybereagle2001*/

import algorea.Scanner;

class Main{
   static Scanner input= new Scanner(System.in);
   public static void main(String[] args){
      int age= input.nextInt();
      int poids= input.nextInt();
      if (age == 60){
         System.out.println(0);
      }
      else if (age < 10){
         System.out.println(5);
      }
      else{
         if (poids >= 20){
            System.out.println(40);
         }
         else{
            System.out.println(30);
         }
      }
   }
}
